htslib/vcf.h

```diff
@@ -517,6 +517,25 @@ set to one of BCF_ERR* codes and must be checked before calling bcf_write().
     HTSLIB_EXPORT
     int vcf_write_line(htsFile *fp, kstring_t *line);
 
+
+
+    /// Write a line to a VCF file for indexing
+    /** @param line   Line to write
+        @param fp     File to write it to
+        @param h      The header for the vcf file
+        @param chr_id Chromosome id
+        @param name   Chromosome name
+        @param beg    Beginning position
+        @param end    End position
```

Comment:
The parameters do not match the function declaration.  It has pos & len while this has beg & end.  Also we should probably list them in the same order, so move "line" down after the header.